lib/burnside/intrinsics.cc

```diff
@@ -358,6 +359,8 @@ std::string IntrinsicLibrary::Implementation::getWrapperName(Context &c) {
   } else if (auto cplx{resultType.dyn_cast<fir::CplxType>()}) {
     // TODO using kind here is weird, but I do not want to hard coded mapping
     return prefix.str() + c.name.str() + ".c" + std::to_string(cplx.getFKind());
+  } else if (auto firf{resultType.dyn_cast<fir::RealType>()}) {
+    return prefix.str() + c.name.str() + ".f" + std::to_string(firf.getFKind());
```

Comment:
This looks like a good place to use `llvm::Twine`.

Reply:
Thanks, I did not know about this. I have changed the code to use it and removed parts that were duplicated.